site/src/pages/components/customizableselect.mdx

```diff
@@ -62,18 +62,16 @@ And here the same `<select>` with opt ins to the new stylability behavior which
 
 ## HTML parser changes
 
-The current behavior in the HTML parser for `<select>` is to remove all tags which aren't `<option>` or `<optgroup>`. We will change the HTML parser to allow `<datalist>` and `<button>` as a child of `<select>`, and allow any other content within that `<datalist>` or `<button>`. The `<button>` allows authors to replace the in-page button which opens the popup, and the `<datalist>` allows authors to replace the popup element containing the options.
+The current behavior in the HTML parser for `<select>` is to remove all tags which aren't `<option>` or `<optgroup>`. We will change the HTML parser to allow almost all tags as descendants of `<select>`. Adding `<button>` as a child of `<select>` allows authors to replace the in-page button which opens the popup, and all other tags are put in the popup element containing the options.
```

Comment:
"put in the popup element" sounds like they're reparented during parsing. Perhaps "all other tags are slotted into the popover element" or something like that?

Reply:
makes sense, done